### PR TITLE
Create python-api-walkthrough.md

### PR DESCRIPTION
**Pull request by aman-ebay**
This Google Cloud Shell walkthrough is linked to Cloud Dataproc documentation to be published at: https://cloud.google.com/dataproc/docs/tutorials/python-library-example (cl/210029919)

### CODE REVIEW

File: dataproc/python-api-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    * <walkthrough-enable-apis apis="dataproc,compute_component,storage-component.googleapis.com"></walkthrough-enable-apis>
+
+## Prerequisites (2)
+
+1. This walkthrough uploads a PySpark file (`pyspark_sort.py`) to a
+   [Cloud Storage bucket](https://cloud.google.com/storage/docs/key-terms#buckets) in
+   your project.
+   * You can use the [Cloud Storage browser page](https://console.cloud.google.com/storage/browser)
+   in Google Cloud Platform Console to view existing buckets in your project.
+   * To create a new bucket, run the following command. Your bucket name must be unique.
```
Comment (tswast) [inline dataproc/python-api-walkthrough.md]: Could we add a big "OR" to this? I mindlessly pasted the command to create a bucket when I already have one that I like to use for tutorials.

File: dataproc/python-api-walkthrough.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    --gcs_bucket=$BUCKET \
+    --create_new_cluster
+    ```
+
+## Job Output
+
+Job output in Cloud Shell shows cluster creation, job submission,
+    job completion, and then tear-down of the cluster.
+
+     ```bash
```
Comment (tswast) [inline dataproc/python-api-walkthrough.md]: This isn't formatting right in Cloud Shell (I see the "```bash")

File: dataproc/python-api-walkthrough.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+* **View job details from the Console.** View job details by selecting the
+   PySpark job from the Cloud Dataproc 
+   [Jobs page](https://console.cloud.google.com/dataproc/jobs)
+   in the Google Cloud Platform Console.
+
+* **Delete resources used in the walkthrough.**
+   The `submit_job.py` job deletes the cluster that it created for this
+   walkthrough. You can run the following command to delete the
+   Cloud Storage bucket used in this walkthrough (the bucket must be empty).
```
Comment (tswast) [inline dataproc/python-api-walkthrough.md]: Can we suggest deleting the bucket only if they created one for the tutorial? It wouldn't be good to accidentally delete an existing bucket.

File: dataproc/python-api-walkthrough.md (hunk 1 above)
Comment (aman-ebay) [reply dataproc/python-api-walkthrough.md]: Done.

Comment (aman-ebay) [reply dataproc/python-api-walkthrough.md]: Done.

File: dataproc/python-api-walkthrough.md (hunk 2 above)
Comment (aman-ebay) [reply dataproc/python-api-walkthrough.md]: Fixed.

File: dataproc/python-api-walkthrough.md (hunk 3 above)
Comment (aman-ebay) [reply dataproc/python-api-walkthrough.md]: Done.

**Pull request merged**